pgvector/django/forms.py

```diff
@@ -10,3 +10,8 @@ def has_changed(self, initial, data):
         if isinstance(initial, np.ndarray):
             initial = initial.tolist()
         return super().has_changed(initial, data)
+
+    def to_python(self, value):
+        if value in self.empty_values:
```

Comment:
I think `isinstance(value, str) and value == ''` would be a better check so `[]` is still treated as a vector. Also, please keep the trailing newline.